ANS_JS_template.tex

```diff
@@ -188,116 +188,64 @@ \subsection{Other Research on NRHES Models}
-
-Due to the uncertainties, NFCSs are best used as comparative tools, as opposed to predicting absolute values. Given different constraints within an NFCS, it can demonstrate the relative differences between the two simulations better than the absolute nature of either one. The strength of relative comparisons is especially true in economic analyses, where the true prices and how they are likely to change over time are unpredictable. The simulator, if built well, would be able to show which model would be cheaper, not the exact price associated with it  \cite{Wilson2012}. Some of the uncertainty concerns surrounding simulating a nuclear fuel cycle as stated by Prof. Wilson:
+The major difference between many of these nuclear fuel cycle simulators is the ability to model discrete facilities and fuel batches versus modeling continuous mass flows throughout the fleet of facilities operating in an identical fashion. In the case of the fleet modeling, the average value for each of the facilities is assumed. The discrete model requires more information while providing a more detailed analysis. The discrete model is, however, more computationally complex and expensive.
 
-\begin{itemize}
-\item Cost of water
-\item Uranium consumption and costs
-\item Total land use
-\item Waste disposal space
-\item Localized variations of the estimated peak dose rate
-\item Radiotoxicity of waste streams
-\item Radioactive liquid and gaseous release volumes
-\item Conventional pollutant release over the lifecycle of the nuclear system
-\item Required Enrichment of the fuel
-\item Attractiveness of separated materials for proliferation of nuclear weapons
-\item Difficulties in siting a nuclear power plant, and
-\item Variation in how metrics are calculated.
-\end{itemize}
+In "Modeling the Nuclear Fuel Cycle," Juchau et al. (2010) describe four functions as necessary for a NFCS to meet the desired qualities of an organization proposed at the time called the Simulation Institute for Nuclear Enterprise Modeling and Analysis \cite{Juchau2010}. The four functions, which are reiterated in other synopsis of Nuclear Fuel Cycle Simulators \cite{Wilson2012, NEA2012} include: discrete facilities and material tracking, uncertainty analyses, ability to optimize, and open and accessible with high quality documentation to ensure collaboration. Juchau et al. (2010) reviews the nuclear fuel cycle simulator tools CAFCA, COSI, CEPMNFC, DANESS, NFCSim, VEGAS, VISION, NFCSS, and GENIUS 1. His conclusion is that many of these tools have overlapping functionalities, partially due to the proprietary nature of at least parts of each.
```

Comment:
Added citation to Wilson and the NEA article as having some similar conclusions as Juchau et. al.